lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -507,13 +507,12 @@ defmodule Kernel.SpecialForms do
   defmacro alias(module, opts), do: error!([module, opts])
 
   @doc """
-  Requires a given module to be compiled and loaded.
+  Tells Elixir you want to use the macros defined in a module.
 
   ## Examples
 
-  Notice that usually modules should not be required before usage,
-  the only exception is if you want to use the macros from a module.
-  In such cases, you need to explicitly require them.
+  Exported module functions are globally available, but in order to use
```

Comment:
"Exported module functions" reads somehow a bit clunky to me personally. What if we go with

> All the functions that a module exports are available everywhere, but in order to use [same as before]

Thoughts?